src/Platforms/DB2Platform.php

```diff
@@ -25,6 +25,25 @@
 
 class DB2Platform extends AbstractPlatform
 {
+    /** @see https://www.ibm.com/docs/en/db2/11.5?topic=views-syscatcolumns */
+    public const SYSCAT_COLUMNS_GENERATED_DEFAULT = 'D';
+
+    /** @see https://www.ibm.com/docs/en/db2/11.5?topic=views-syscatindexes */
+    public const SYSCAT_INDEXES_UNIQUERULE_PERMITS_DUPLICATES     = 'D';
+    public const SYSCAT_INDEXES_UNIQUERULE_IMPLEMENTS_PRIMARY_KEY = 'P';
+
+    /** @see https://www.ibm.com/docs/en/db2/11.5?topic=views-syscattabconst */
+    public const SYSCAT_TABCONST_TYPE_PRIMARY_KEY = 'P';
+
+    /** @see https://www.ibm.com/docs/en/db2/11.5?topic=views-syscatreferences */
+    public const SYSCAT_REFERENCES_UPDATERULE_RESTRICT = 'R';
+    public const SYSCAT_REFERENCES_DELETERULE_CASCADE  = 'C';
+    public const SYSCAT_REFERENCES_DELETERULE_SET_NULL = 'N';
+    public const SYSCAT_REFERENCES_DELETERULE_RESTRICT = 'R';
```

Comment:
I think this way we could provide tests in a easier way. I'm not against in making these private if you think it is better.

Reply:
Introducing those constants to help the reader understand those magic values is a good idea imho. But I'd like to avoid introducing a public API that I need to maintain. So declaring them `private` would be my preference.

Reply:
It's fine for me too. They were converted to private.